tdrs-backend/gunicorn_start.sh

```diff
@@ -12,11 +12,9 @@ else
     ( cd  /home/vcap/deps/0/bin/; ./redis-server /home/vcap/app/redis.conf &)
 fi
 
-#
-echo "Applying database migrations"
-python manage.py makemigrations
-python manage.py migrate
-python manage.py populate_stts
+# python manage.py makemigrations
+# python manage.py migrate
+# python manage.py populate_stts
```

Comment:
In testing steps of this PR, please provide the process we discussed during dev sync and/or ping me with relevant draft so we can be prepared for getting this fix to Alex.